client/packages/uml-theia-integration/src/browser/command-contribution.ts

```diff
@@ -51,24 +51,25 @@ export class UmlModelContribution implements CommandContribution, MenuContributi
                 if (this.workspaceService.tryGetRoots().length) {
                     workspaceUri = new URI(this.workspaceService.tryGetRoots()[0].uri);
 
-                    this.showInput("Name", "Name of UML Diagram", nameOfUmlModel => {
-                        this.showInput("Diagram type", "Type of UML Diagram (activity | class | package | sequence | statemachine | usecase)", diagramType => {
-                            this.createUmlDiagram(nameOfUmlModel, workspaceUri, diagramType);
-                        });
+                    this.showInput("Enter Name of UML Diagram", "Diagram name", nameOfUmlModel => {
+                        this.showInput("Enter UML Diagram Type", "activity | class | component | deployment | package | sequence | statemachine | usecase",
+                            diagramType => {
+                                this.createUmlDiagram(nameOfUmlModel, workspaceUri, diagramType);
+                            });
```

Comment:
Maybe more concise if you'd inline this block (not using {}), but that's just a style nitpick

```diagramType => this.createUmlDiagram(nameOfUmlModel, workspaceUri, diagramType)```